api_tests/src/post.spec.ts

```diff
@@ -651,8 +652,17 @@ test("Enforce community ban for federated user", async () => {
   );
   expect(unBanAlpha.banned).toBe(false);
 
-  // Need to re-follow the community
-  await followBeta(alpha);
+  // Check that unban was federated to alpha
+  await waitUntil(
+    () => getModlog(alpha),
+    m => {
+      console.log(JSON.stringify(m.modlog[0], null, 4));
```

Comment:
Could probably remove this logging line now that its passing.

Reply:
Right, done